include/RAJA/util/macros.hpp

```diff
@@ -141,9 +141,10 @@ RAJA_HOST_DEVICE RAJA_INLINE void RAJA_UNUSED_VAR(T&&...) noexcept
  */
 #if defined(RAJA_ENABLE_OPENMP)
 #define RAJA_OMP_DECLARE_REDUCTION_COMBINE                                     \
+  RAJA_UNUSED_VAR(EXEC_POL {});                                                \
```

Comment:
Is this needed?

Reply:
Without this you get lots of warnings about an unused local alias, probably because the alias is only used in the pragma.